static/js/dropdown_list_widget.js

```diff
@@ -77,8 +77,14 @@ const DropdownListWidget = function (opts) {
             `#${CSS.escape(opts.container_id)} .dropdown-search > input[type=text]`,
         );
         const dropdown_toggle = $(`#${CSS.escape(opts.container_id)} .dropdown-toggle`);
+        const get_data = (value) => {
+            if (!value || value === opts.null_value) {
+                return opts.data;
+            }
+            return opts.data.filter((x) => x.value !== opts.value.toString());
+        };
```

Comment:
Will this have an effect on other places we use this widget?  @aero31aero thoughts would be appreciated for whether this is a good idea or something that should be controlled as an option.

Reply:
Yep, it will affect other places as well.
If the fact of excluding the current list item from the dropdown doesn't seem right, is it better to have an extra object (parameter) while passing it to `dropdown_list_widget`.

Something like: 
(Will exclude the current list item from the dropdown.)
```javascript
const opts = {
  retain_current_item: false,
};

dropdown_list_widget(opts);
```

having a default value of `true`.

Reply:
Yeah, let's do that parameter for now - that way we can contain the complexity of this change, and then please start a thread on #frontend about whether we want to change the value of that parameter for other users of this widget.

Reply:
Cool! Will refactor it.